src/Tizen.Uix.Sticker/Tizen.Uix.Sticker/StickerConsumer.cs

```diff
@@ -0,0 +1,537 @@
+        private static StickerConsumerEventCallback _eventDelegate;
+
+        /// <summary>
+        /// Gets a flag indicating whether the StickerConsumer is initialized
+        /// </summary>
+        public static bool Initialized { get; private set; }
+
+        /// <summary>
+        /// Initialize sticker consumer.
+        /// </summary>
+        /// <since_tizen> 10 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/mediastorage</privilege>
+        /// <privlevel>public</privlevel>
+        /// <feature>http://tizen.org/feature/ui_service.sticker</feature>
+        /// <exception cref="NotSupportedException">The required feature is not supported.</exception>
+        /// <exception cref="UnauthorizedAccessException">This exception can be due to permission denied.</exception>
+        /// <exception cref="InvalidOperationException">
+        /// This can occur due to the following reasons:
+        /// 1) This exception can be due to out of memory.
+        /// 2) This exception can be due to operation failed.
+        /// </exception>
+        public static void Initialize()
+        {
+            if (!Initialized)
+            {
+                IntPtr handle;
+                ErrorCode error = StickerConsumerCreate(out handle);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Create Failed with error " + error);
+                    throw ExceptionFactory.CreateException(error);
+                }
+                _handle = handle;
+                Initialized = true;
+            }
+            else
+            {
+                Log.Debug(LogTag, "Already initialized");
+            }
+        }
+
+        /// <summary>
+        /// Deinitialize the sticker consumer.
+        /// </summary>
+        /// <since_tizen> 10 </since_tizen>
+        /// <feature>http://tizen.org/feature/ui_service.sticker</feature>
+        /// <exception cref="NotSupportedException">The required feature is not supported.</exception>
+        /// <exception cref="InvalidOperationException">This exception can be due to operation failed.</exception>
+        /// <pre>
+        /// StickerConsumer must be initialized.
+        /// </pre>
+        public static void Deinitialize()
+        {
+            ErrorCode error = StickerConsumerDestroy(_handle);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "Destroy Failed with error " + error);
+                if (error == ErrorCode.InvalidParameter)
+                    throw ExceptionFactory.CreateException(ErrorCode.OperationFailed);
+                else
+                    throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Retrieves all sticker data in the sticker database.
+        /// If you set the @a offset as @c 10 and @a count as @c 10, then only retrieved data from @c 10 to @c 19.
```

Comment:
the same in other copies of this line: 151, 181, 211, 241.
```suggestion
        /// If you set the <paramref name="offset"/> as 10 and <paramref name="count"/> as 10, then only records from 10 to 19 will be retrieved.
```
having offset and count gives us a function which only returns part of the data, but description says "all sticker data".  But that's not very important.

Reply:
I fixed it. please check.